modules/manually-maintained-credentials-upgrade.adoc

```diff
@@ -8,20 +8,50 @@
 [id="manually-maintained-credentials-upgrade_{context}"]
 = Upgrading clusters with manually maintained credentials
 
-If credentials are added in a future release, the Cloud Credential Operator (CCO) `upgradable` status for a cluster with manually maintained credentials changes to `false`. For minor release, for example, from 4.7 to 4.8, this status prevents you from upgrading until you have addressed any updated permissions. For z-stream releases, for example, from 4.7.12 to 4.7.13, the upgrade is not blocked, but the credentials must still be updated for the new release.
+The Cloud Credential Operator (CCO) `upgradable` status for a cluster with manually maintained credentials is `false` by default.
 
-Use the *Administrator* perspective of the web console to determine if the CCO is upgradeable.
+* For minor releases, for example, from 4.7 to 4.8, this status prevents you from upgrading until you have addressed any updated permissions and changed the `upgradable` status to `true`.
 
-. Navigate to *Administration* -> *Cluster Settings*.
+* For z-stream releases, for example, from 4.7.12 to 4.7.13, the upgrade is not blocked, but the credentials must still be updated for the new release.
```

Comment:
@joelddiaz do we expect permissions update in credentials request for z-stream releases? I assumed the answer in no and that is why we do not have `upgradable = false` guard here